unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PluginSystem/PluginSystemFactory.cs

```diff
@@ -83,6 +84,9 @@ public static PluginSystem Create()
 
             pluginSystem.Register<MapTexturePlugin>(() => new MapTexturePlugin());
 
+            pluginSystem.RegisterWithFlag<BackpackEditorV2Plugin>(() => new BackpackEditorV2Plugin(), "backpack_editor_v2");
+            pluginSystem.RegisterWithFlag<AvatarEditorHUDPlugin>(() => new AvatarEditorHUDPlugin(), "backpack_editor_v1");
```

Comment:
We could add some kind of TODO comment in this line in order to inform that it will be something temporal until we deprecate the old Backpack.

Reply:
done